modules/nickname/events/guildMemberUpdate.js

```diff
@@ -0,0 +1,68 @@
+const {truncate} = require("../../../src/functions/helpers");
+module.exports.run = async function (client, oldGuildMember, newGuildMember) {
+
+    if (!client.botReadyAt) return;
+    if (newGuildMember.guild.id !== client.guild.id) return;
+    if (newGuildMember.nickname === oldGuildMember.nickname && newGuildMember.roles.highest.position === oldGuildMember.guild.me.roles.highest.position) return;
+
+    const roles = client.configurations['nickname']['config'];
+    const moduleModel = client.models['nickname']['User'];
+
+    let rolePrefix = "";
```

Comment:
```suggestion
```
declaration not required if [other review](https://github.com/ScootKit/CustomDCBot/pull/142/files#r1643483368) commit is commited, otherwise please ignore.